compiler/one-cmds/optimization/ODEFAULT.cfg

```diff
@@ -0,0 +1,99 @@
+# These options copied from 'onelib/constant.py`
+[one-optimize]
+; Experimental: This will convert NCHW operators to NHWC under the assumption that input model is NCHW.
+convert_nchw_to_nhwc=True
+; expand broadcastable constant node inputs
+expand_broadcast_const=True
+; onvert the input shape of the model (argument for convert_nchw_to_nhwc)
+nchw_to_nhwc_input_shape=True
+; convert the output shape of the model (argument for convert_nchw_to_nhwc)
+nchw_to_nhwc_output_shape=True
```

Comment:
These are used when source is ONNX file. I don't think these should be in `DEFAULT`.
I wonder what meaning of `DEFAULT` is...